.github/workflows/submit-gradle-dependencies.yml

```diff
@@ -20,5 +20,5 @@ jobs:
         uses: gradle/gradle-build-action@v2
         with:
           dependency-graph: generate-and-submit
```

Comment:
Warning (⚠️): You are not removing these 3 lines above, on `gradle/gradle-build-action@v2` and `generate-and-submit`, just like you did on the [WCAndroid#11950](https://github.com/woocommerce/woocommerce-android/pull/11950/files) PR, was that maybe an oversight? 🤔

Reply:
Ah yes, thank you!